frontend/src/vault/keygen/shared/KeygenBackup.tsx

```diff
@@ -25,6 +25,8 @@ const Wrapper = styled.div`
 export const KeygenBackup = () => {
   const { t } = useTranslation();
   const navigate = useNavigate();
+  const location = useLocation();
```

Comment:
I think we might need better ways to determinate the vault whether it is fast or not , you can check whether there is a signer with `Server-*` in it